modules/nw-ovn-kubernetes-get-raft-db-status.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-querying-status-northbound-southbound-databases_{context}"]
+= Querying the status of the northbound and southbound databases
+
+The Kubernetes control plane nodes each contain an `ovnkube-master` pod which hosts containers for the OVN northbound and southbound databases which must form `Raft` consensus between its peer database. For example all OVN northbound databases must form a `Raft` cluster and all southbound databases must form a separate `Raft` cluster. The OVN northbound and southbound databases must have distinct `Raft` leaders.
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (oc) installed.
+
+.Procedure
+
+. Run the following command to get all resources in the OVN-Kubernetes project:
++
+[source,terminal]
+----
+$ oc get all -n openshift-ovn-kubernetes
```

Comment:
Already showed how to get all the resources in the "Listing all resources in the OVN-Kubernetes project" above.  I feel this command is not necessary.

Reply:
@CFields651 I guess I feel maybe user will not read doc in linear fashion and might just see this section in TOC so they will simply go to this section, so I was keeping it all self-contained. Personally if I go to a section I like for the most part having everything I need there within reason